eqc/mstore_eqc.erl

```diff
@@ -44,7 +42,7 @@ delete(FileSize, Size) ->
 
 store(FileSize, Size) ->
     ?LAZY(oneof(
-            [{{call, ?MODULE, new, [FileSize, ?DIR]}, {call, ?G, empty, []}}
+            [{{call, ?MODULE, new, [FileSize, ?DIR]}, {call, gb_trees, empty, []}}
```

Comment:
Line 45 is too long:             [{{call, ?MODULE, new, [FileSize, ?DIR]}, {call, gb_trees, empty, []}}.